prow/test/integration/config/prow/cluster/fakepubsub.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: default
+  name: fakepubsub
```

Comment:
I really want to stay consistent with the existing fake services named fake*. If we do want to change it to `pubsub-emulator` we should also rename fakegitserver to just `gitserver` because it's not even an emulator (it's the real deal).

But yeah it's so much nicer to see "fake*" in the codebase when writing tests and configuring things because of the consistency IMHO.